src/components/patients/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Header from "components/header";
+import PatientTable from "components/patients/patientTable";
+import SearchTool from "components/patients/searchTool";
+import "./style.css";
+
+const Patients = () => {
+  return (
+    <>
+      <Header />
+      <div className="homeSearch">
+        <div className="searchTool bp3-card bp3-elevation-3">
```

Comment:
I don't think you should add the classNames coming from Blueprint by hand, are you sure you dont want to use the `Card` component from Blueprint ?